src/components/pages/Photos/elements/roverNavbar/RoverNavbar.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import RoverButton from './RoverButton';
+import './RoverNavbar.css';
+
+export default function Navbar() {
+  return (
+    <ul className="rover-navbar">
+      <li className="rover-box">
+        <RoverButton roverName="curiosity" />
+        <RoverButton roverName="Opportunity" />
+        <RoverButton roverName="Spirit" />
+      </li>
+    </ul>
```

Comment:
I'm unsure about this list structure... could probably just be a `<div>` with `<RoverButton>` child elements?